packages/apollo-server-express/src/ApolloServer.test.ts

```diff
@@ -73,6 +73,60 @@ describe('apollo-server-express', () => {
       expect(result.errors, 'errors should exist').not.to.exist;
     });
 
+    it('can enable gui separately from instrospection during production', async () => {
```

Comment:
this test seems backwards: enabling gui without introspection seems worthless, whereas enabling introspection without gui (for a different gui) seems useful?